app/views/layouts/decidim/_logo.html.erb

```diff
@@ -1,3 +1,3 @@
-<div class="hki-city-logo">
-  <%= image_tag "helsinki/helsinki-logo-black.svg", alt: "Helsinki" %>
+<div class="seattle-logo">
```

Comment:
ℹ️ I'm not sure what element we should be using here. h1? Something else? I haven't done html in a while.